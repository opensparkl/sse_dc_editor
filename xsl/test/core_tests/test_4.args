--stringparam id deep_in_rubbish