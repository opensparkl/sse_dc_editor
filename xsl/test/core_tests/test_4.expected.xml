<?xml version="1.0"?>
<errors>
  <error message="invalid_parent" value="complete" help="trbl_err_invalid_parent" context="consume" ref="deep_in_rubbish"/>
  <warning message="external_field" value="field1" help="trbl_warn_external_role" context="consume" ref="deep_in_rubbish"/>
  <warning message="external_service" value="Service" help="trbl_warn_external_service" context="consume" ref="deep_in_rubbish"/>
  <hint message="missing_attribute" value="value" context="consume" ref="deep_in_rubbish"/>
  <hint message="missing_prop" value="instance.bind" help="api_instance_bind" context="consume" ref="deep_in_rubbish">
    <msg key="instance-bind">
        (Optional) Bind instance.spec Vars to operation field values.
      </msg>
  </hint>
</errors>
